Gemfile

```diff
@@ -1,7 +1,6 @@
 source "https://rubygems.org"
 
+gem "kitchen-docker", ">= 2.9"
+gem "kitchen-salt", ">=0.5.0"
```

Comment:
`0.6.0` available now, which fixes the `kitchen-salt` issue.

Reply:
@myii thanks for this info